problems/enumerated/problem_10384/problem_10384.py

```diff
@@ -0,0 +1,306 @@
+from manim import *
+from lib.aramanim import *
+from objects import Scissors
+from .text import *
+
+class Problem10384(Scene):
+	def construct(self):
+		self.wait()
+		self.set_up()
+
+		self.play(Write(self.task_number))
+
+		self.wait()
+
+		self.play(Write(self.condition1))
+		self.wait()
+		self.play(Write(self.condition2))
+		self.wait(1)
+
+		self.play(
+            VGroup(self.condition1, self.condition2).animate.scale(.8).to_edge(DOWN+LEFT)
+        )
+
+		self.wait()
+
+		self.play(Write(self.first_rope_label))
+		self.play(GrowFromEdge(self.first_rope, LEFT))
+		self.play(Write(self.second_rope_label))
+		self.play(GrowFromEdge(self.second_rope, LEFT))
+
+		# կտրել հերթով
+		# self.scissors1.cut(self)
+		# self.first_rope.become(self.get_part(2.5, color=YELLOW).align_to(self.first_rope, LEFT+UP))
+		# self.part2_of_first_rope.next_to(self.first_rope, buff=0)
+		# self.add(self.part2_of_first_rope)
+		# part2_of_first_rope_label = Tex(part2_of_first_rope_string, font_size=40)
+		# part2_of_first_rope_label.add_updater(
+		# 	lambda x: x.next_to(self.part2_of_first_rope, UP)
+		# )
+		# self.play(FadeIn(part2_of_first_rope_label))
+		# self.scissors1.fade_out(self)
+		# self.play(self.part2_of_first_rope.animate.shift(RIGHT*3))
+
+		# #self.play(FadeOut(self.part2_of_first_rope))
+
+		# self.scissors2.cut(self)
+		# self.second_rope.become(self.get_part(5).align_to(self.second_rope, LEFT+UP))
+		# self.part2_of_second_rope.next_to(self.second_rope, buff=0)
+		# self.add(self.part2_of_second_rope)
+		# part2_of_second_rope_label = Tex(part2_of_first_rope_string, font_size=40)
+		# part2_of_second_rope_label.add_updater(
+		# 	lambda x: x.next_to(self.part2_of_second_rope, UP)
+		# )
+		# self.play(FadeIn(part2_of_second_rope_label))
+		# self.scissors2.fade_out(self)
+		# self.play(self.part2_of_second_rope.animate.shift(RIGHT*3))
+
+		#self.play(FadeOut(self.part2_of_second_rope))
+
```

Comment:
If code is not necessary, it will be better to delete it.
Commented code should not be in final code.